build.gradle

```diff
@@ -33,6 +38,17 @@ subprojects {
         testCompile 'junit:junit:4.+'
     }
 
+    jacocoTestReport {
+        additionalSourceDirs = files(sourceSets.main.allSource.srcDirs)
+        sourceDirectories = files(sourceSets.main.allSource.srcDirs)
```

Comment:
Are these lines necessary? According to code (https://github.com/gradle/gradle/blob/master/subprojects/jacoco/src/main/groovy/org/gradle/testing/jacoco/tasks/JacocoReport.groovy#L195 ), JacocoReport uses `allJava.getSrcDirs()` as the default setting for `sourceDirectories`.